bot/resources/halloween/spooky_rating.json

```diff
@@ -0,0 +1,47 @@
+{
+    "0": {
```

Comment:
You'll have to make this -1 due to how `bisect` works or change the implementation, this bug is present in the other command too I just haven't gotten around to fixing it (yet).

I hope this example from the [bisect documentation page](https://docs.python.org/3/library/bisect.html#other-examples) illustrates the issue:
```py
>>> def grade(score, breakpoints=[60, 70, 80, 90], grades='FDCBA'):
...     i = bisect(breakpoints, score)
...     return grades[i]
...
>>> [grade(score) for score in [33, 99, 77, 70, 89, 90, 100]]
['F', 'A', 'C', 'C', 'B', 'A', 'A']
```